org.envirocar.app/src/org/envirocar/app/views/login/SigninActivity.java

```diff
@@ -155,22 +160,22 @@ protected void onLoginClicked() {
         View focusView = null;
 
         // Reset errors
-        this.usernameEditText.setError(null);
-        this.passwordEditText.setError(null);
+        this.usernameEditTextLayout.setError(null);
+        this.passwordEditTextLayout.setError(null);
 
         // Store values at the time of the login attempt
         String username = usernameEditText.getText().toString().trim();
         String password = passwordEditText.getText().toString();
 
         // check for valid password
-        if (password == null || password.isEmpty() || password.equals("")) {
-            this.passwordEditText.setError(getString(R.string.error_field_required), errorPassword);
+        if (password == null || password.isEmpty()) {
```

Comment:
``password.equals("")`` was removed cause ``password.isEmpty()`` was already checking the same here.